server/api.py

```
@@ -0,0 +1,312 @@
+
+agent: Agent()
+
+def get_or_load_model(model_spec: str, verbose: bool = False) -> MLXRunner:
+    """Get model from cache or load it if not cached."""
+    global _model_cache, _current_model_path
+
+    print(model_spec)
+    # Use the existing model path resolution from cache_utils
+
+    try:
+        model_path, model_name, commit_hash = get_model_path(model_spec)
```

Ruff (0.14.1)

Unpacked variable `commit_hash` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)